### ISSUE TITLE
ChipSet errors when rendered without children

### ISSUE THREAD

**Issue by lucasecdb** [material-components/material-components-web-react#880]
The following error occurs if the `ChipSet` component renders without any children:

![image](https://user-images.githubusercontent.com/10223856/58301800-639cf600-7dbe-11e9-94c9-547a49d9bc0e.png)

**Comment by lucasecdb** [comment]
actually, it's not that when it's rendered without children, but when rendered with conditional chips, like

```jsx
<ChipSet>
  {myCondition && <Chip label="hello" />}
  {thisIsNull && <Chip label="world" />}
</ChipSet>
```

**Comment by moog16** [comment]
Definitely a bug. I'll try looking at this today.

**Comment by moog16** [comment]
not going to be able to get to it today :(

**Comment by lucasecdb** [comment]
no problem! I have a workaround while this isn't resolved 😄

I may look into it this weekend

**Comment by moog16** [comment]
That'd be great!

**Comment by moog16** [comment]
Closed with #883
